client/src/components/Logout/Logout.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react";
+import AuthService from "../../Services/AuthService";
+// import { AuthContext } from "../Context/AuthContext";
+
+
+
+function Logout(props) {
+  // const { isAuthenticated, user, setIsAuthenticated, setUser } = useContext(AuthContext);
+
+  const logoutHandler = () => {
+    AuthService.logout().then((data) => {
+      console.log(data);
+      if (data.success) {
+        // setUser(data.user);
+        // setIsAuthenticated(false);
+        console.log(data.success);
```

Comment:
Do we want to display a message telling the user they have logged out? rather than console logging

Reply:
i will clear with next pull request, used for testing.

Reply:
Code to be used later. Console log to be removed

Reply:
cleaned the code